coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/budget/core/ProposalBudgetHierarchyController.java

```diff
@@ -0,0 +1,72 @@
+package org.kuali.coeus.propdev.impl.budget.core;
```

Comment:
Can move it, but unless we create a new budget service it will be the only thing in it.